deadmatterconfig.json

```diff
@@ -0,0 +1,128 @@
+        "FieldName":"SteamServerName",
+        "InputType":"text",
+        "IsFlagArgument":false,
+        "ParamFieldName":"SteamServerName",
+        "IncludeInCommandLine":true,
+        "DefaultValue":"AMP Powered Server",
+        "EnumValues":{}
+    },
+    {
+        "DisplayName":"Lobby Header",
+        "Category":"Dead Matter Server Settings",
+        "Description":"Sets the server title displayed in the lobby",
+        "Keywords":"server,name,lobby,servername",
+        "FieldName":"ServerName",
+        "InputType":"text",
+        "IsFlagArgument":false,
+        "ParamFieldName":"/Script/DeadMatter.DMGameSession.ServerName",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"AMP Powered Server",
+        "EnumValues":{}
+    },
+    {
+        "DisplayName":"MOTD",
+        "Category":"Dead Matter Server Settings",
+        "Description":"Sets the server Message of the Day (MOTD)",
+        "Keywords":"server,motd,welcome,message,day",
+        "FieldName":"MOTD",
+        "InputType":"text",
+        "IsFlagArgument":false,
+        "ParamFieldName":"/Script/DeadMatter.DMGameSession.MOTD",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"This server is powered by AMP",
+        "EnumValues":{}
+    },
+    {
+        "DisplayName":"Player Limit",
+        "Category":"Dead Matter Server Settings",
+        "Description":"Sets the maximum number of players allowed on the server",
+        "Keywords":"max,players,limit,maxplayers",
+        "FieldName":"$MaxUsers",
+        "InputType":"number",
+        "MinValue":"1",
+        "IsFlagArgument":false,
+        "ParamFieldName":"MaxPlayers",
+        "IncludeInCommandLine":true,
+        "DefaultValue":"32",
+        "Placeholder":"32",
+        "Suffix":"players",
+        "EnumValues":{}
+    },
+    {
+        "DisplayName":"Enable PvP",
+        "Category":"Dead Matter Server Settings",
+        "Description":"If set, PvP will be enabled",
+        "Keywords":"pve,pvp",
+        "FieldName":"PVP",
+        "InputType":"checkbox",
+        "IsFlagArgument":false,
+        "ParamFieldName":"/Script/DeadMatter.DMGameSession.PVP",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"true",
+        "EnumValues":{
+            "False":"false",
+            "True":"true"
+        }
+    },
+    {
+        "DisplayName":"Additional Command Line Arguments",
+        "Category":"Dead Matter Server Settings",
+        "Description":"Sets additional command arguments to start the server with that are not otherwise set by AMP",
+        "Keywords":"switches,command,line,arguments",
+        "FieldName":"ServerSwitches",
+        "InputType":"text",
+        "IsFlagArgument":false,
+        "ParamFieldName":"ServerSwitches",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"",
+        "Placeholder":"-hardcore -heartbeat",
+        "EnumValues":{}
+    },
+    {
+        "DisplayName":"Admin",
```

Comment:
This Admin setting, and the Super Admin setting, don't work as desired. As I understand it, the Game.ini can include multiple Admin and SuperAdmin entries. However, these settings will overwrite all of them to the same value.

Reply:
Yes, you can't do multiple in AMP (a long requested feature). You will have to leave users to include the lines manually - which is better than them only being able to include one Admin or one SuperAdmin

Reply:
In some other templates for situations like these, I have included an "info box" in the UI to tell the user how to specify the relevant setting manually

Reply:
Changed in latest commit